src/Abp/Runtime/Caching/AbpCacheBase.cs

```diff
@@ -32,42 +32,42 @@ protected AbpCacheBase(string name) : base(name)
 
         public virtual TValue Get(TKey key, Func<TKey, TValue> factory)
         {
-            TValue item = default(TValue);
+            TValue value = default;
```

Comment:
Why? Then we should rename `items` to `values`.

Reply:
`values` sounds good.